tests/unit/meta/test_meta.py

```diff
@@ -1110,6 +1110,48 @@ def test_snap_hooks_overwrite_part_hooks(self):
             FileContains("from snap"),
         )
 
+    def test_hooks_with_command_chain_write_stub(self):
+        self.config_data["hooks"] = {
+            "configure": {"command-chain": ["fake-hook-command-chain"]}
+        }
+
+        self.generate_meta_yaml()
+
+        self.assertThat(os.path.join(self.hooks_dir, "configure"), FileExists())
```

Comment:
This matches surrounding code, but i thought we were moving away from testtools...? Should new tests be written in it?
something like:
```
configure_hook = pathlib.Path(self.hooks_dir, "config")

assert configure_hook.read_text() == "#!/bin/sh\n"
```

OK either way, just trying to be consistent :+1:

Reply:
consistency, yes